SECURITY.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Security Policy"
+linkTitle: "Security policy"
+weight: 10
+type: "docs"
+aliases:
+  - /contributing/security-policy/
+---
+
+# Knative security policy
+
+## Introduction
+
+Security is core to our values, and we value the input of developers, operators,
+security researchers, and others acting in good faith to help us maintain a high
+standard for the security and privacy for our users, and their users. This
+includes encouraging responsible vulnerability research and disclosure. This
+policy sets out our definition of good faith in the context of finding and
+reporting vulnerabilities, as well as what you can expect from us in return.
+
+_If you have identified a vulnerability in Knative, please do not post an issue
+in the public issue tracker. Instead, please follow the responsible disclosure
+policy described in this document._
+
+## Expectations
+
+When working with us according to this policy, you can expect us to:
+
+- Extend [Safe Harbor](#safe-harbor) for your vulnerability research that is
+  related to this policy;
+- Work with you to understand and validate your report, including a timely
+  initial response to the submission;
+- Work to remediate discovered vulnerabilities in
+  [supported versions](#scope-and-supported-versions) in a timely manner;
+- Publicly disclose accepted vulnerabilities in a timely manner; and
+- Recognize your contribution to improving our security if you are the first to
+  report a unique vulnerability, and your report triggers a code or
+  configuration change.
+
+## Ground Rules
+
+To encourage vulnerability research and to avoid any confusion between
+good-faith hacking and malicious attack, we ask that you:
+
+- Play by the rules. This includes following this policy, as well as any other
+  relevant agreements, particularly the Knative
+  [Code of Conduct](https://knative.dev/community/contributing/code-of-conduct/).
+  If there is any inconsistency between this policy and any other relevant
+  terms, the terms of this policy will prevail;
+- Report any vulnerability you’ve discovered promptly;
+- Avoid violating the privacy of others, disrupting out-of-scope systems,
```

Comment:
My only thought is if 'Avoid' is strong enough word here? Just jotting it down here, but when I think about it, if you accidentally destroy data and hence discover a problem, kind of hard to avoid.

Reply:
Got a suggested re-wording?

Reply:
I played around with doing something about intent vs. accident, and then it would start like this, but then the wording gets a bit wonky:
* Do not intentionally violate the privacy of others
* Do not intentionally disrupt out-of-scope systems
* Do not intentionally destroy data

But, then it gets wordy, so I'm fine with this wording since I can't really come up with anything better. Also it was not a deal stopper, I just jotted it down :)
Fine to resolve by me and keep the wording.